consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/BftForksSchedule.java

```diff
@@ -31,7 +31,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
-public class BftForksSchedule<C extends BftConfigOptions> {
+public class BftForksSchedule<C> {
```

Comment:
Can merge the ForksSchedule refactored code from main now, as I'm sure you know :)